src/batch-client/index.ts

```diff
@@ -233,13 +233,12 @@ function updateAbsoluteFolderPath(originalName: any, parentFolderAbsPath: string
 		if (originalName === 'USER_ROOT') {
 			folder.absFolderPath = `${parentFolderAbsPath}${folder.absFolderPath}`;
 		} else {
-			folder.absFolderPath = folder.absFolderPath.replace(`/${originalName}`, parentFolderAbsPath);
+			folder.absFolderPath = `${parentFolderAbsPath}/${folder?.name}`;
```

Comment:
do we need optional chaining operator here?

Reply:
I have changed the code.